deployment/bare_metal/nginx/sites-available/gradido.conf.template

```diff
@@ -104,15 +98,10 @@ server {
     location /admin {
         limit_req zone=frontend burst=30 nodelay;
         limit_conn addr 40;
-        proxy_http_version 1.1;
-        proxy_set_header   Upgrade $http_upgrade;
-        proxy_set_header   Connection 'upgrade';
-        proxy_set_header   X-Forwarded-For $remote_addr;
-        proxy_set_header   X-Real-IP  $remote_addr;
-        proxy_set_header   Host $host;
-        
-        proxy_pass         http://127.0.0.1:8080/;
-        proxy_redirect     off;
+        rewrite ^/admin/(.*)$ /$1 break;
```

Comment:
Why does the admin-interface needs this `rewrite...`-config and the frontend not?

Reply:
Wie würde ein rewrite vom Frontend aussehen?
rewrite ^/(.*)$ /$1 break;

nginx hängt die Route mit an.
Wenn ich beim admin kein rewrite machen würde,
würde er versuchen die Dateien aus $PROJECT_ROOT/admin/build/admin/
zu lesen.